core/src/main/res/values-de/strings.xml

```diff
@@ -28,6 +29,9 @@
   <!--New episodes fragment-->
   <string name="recently_published_episodes_label">Zuletzt veröffentlicht</string>
   <string name="episode_filter_label">Nur neue Episoden anzeigen</string>
+  <!-- Statistics fragment -->
+  <string name="total_time_listened_to_podcasts">Gesamtzeit gespielter Podcasts:</string>
```

Comment:
But it does not break something if I keep the translations here? Or do I need to delete them again?